docs/pascal_voc_user_manual.md

```diff
@@ -0,0 +1,246 @@
+
+There are few ways to convert Pascal VOC dataset to other dataset format:
+
+``` bash
+datum project import -f voc -i <path/to/voc>
+datum export -f coco -o <path/to/output/dir>
+# or
+datum convert -if voc -i <path/to/voc> -f coco -o <path/to/output/dir>
+```
+
+Also it is possible using filters for converting, check user manual for information about filtering:
+
+``` bash
+datum convert -if voc -i <path/to/voc> \
+    -f yolo -o <path/to/output/dir> \
+    --filter-mode FILTER_MODE \
+    --filter '<xpath filter expression>'
+```
+
+Some formats have extra arguments for converting, run `datum export -f FORMAT -- -h` for more information. For example voc_segmentation format has extra argument `--label_map` which get opportunity to load user color map for segmentation mask:
+
+``` bash
+datum export -f voc_segmentation -- --label-map mycolormap.txt
+
+# mycolormap.txt [label : color_rgb : parts : actions]:
+# cat:0,0,255::
+# person:255,0,0:head:
+```
+
+Also you can use original voc format of label map, for example:
+
+``` bash
+datum export -f voc_layout -- --label-map voc
+```
+
+## Import to Pascal VOC
+There are few ways to convert dataset to Pascal VOC format:
+
+``` bash
+# export dataset into Pascal VOC format (classification) from existing project
+datum export -p <path/to/project> -f voc -o <path/to/export/dir> -- --tasks classification
+
+# converting to Pascal VOC format from other format
+datum convert -if imagenet -i <path/to/imagenet/dataset> \
+    -f voc -o <path/to/export/dir> \
+    -- --label_map voc --save-images
+```
+
+Argument `--tasks` allow to specify tasks for export dataset, by default datumaro uses all tasks.
+Argument   `--label_map` allow to define user label map, for example
+
+``` bash
+# mycolormap.txt [label : color_rgb : parts : actions]:
+# cat:0,0,255::
+# person:255,0,0:head:
+datum export -f voc_segmentation -- --label-map mycolormap.txt
+```
+
+Or you can use original Pascal VOC label map:
+
+``` bash
+datum export -f voc_layout -- --label-map voc
+```
+
+## Datumaro functionality
+Datumaro supports filtering, transformation, merging and etc. for all formats and for the Pascal VOC format in particular. Follow [user manual](../docs/user_manual.md) to get more information about these operations.
+Some examples of using datumaro operations for Pascal VOC dataset format:
+
+``` bash
+# import and compare 2009 and 2012 Pascal VOC datasets
+datum import -f voc <path/to/voc2009> -o ./2009
+datum import -f voc <path/to/voc2012> -o ./2012
+cd 2009 && datum diff ../2012
+...
+Datasets have different lengths: 15674 vs 34314
+Unmatched items in the second dataset: {('2011_002719', 'trainval'), ... }
+
+# extract dataset with only car and bus class items from train subset
+datum filter --mode items+annotations \
```

Comment:
Every transform command outputs result into a new project directory. An output directory can be specified with `-o`.